docs/deploy_features.md

```diff
@@ -19,6 +20,40 @@ We generally find that most developers are able to get high quality answers usin
 * In `infra/main.bicep`, change `chatGptModelName` to 'gpt-4' instead of 'gpt-35-turbo'.
 * You may also need to adjust the capacity above that line depending on how much TPM your account is allowed.
 
+## Using ada-3 embedding models
+
+By default, the deployed Azure web app uses the `text-embedding-ada-002` embedding model. If you want to use one of the ada 3 embedding models, you can do so by following these steps:
```

Comment:
OpenAI never calls the new models "ada 3", they're always called `text-embedding-3`, I don't think ada 3 means anything (unless it's called that in the model). https://openai.com/blog/new-embedding-models-and-api-updates?ref=blog.salesforceairesearch.com

Reply:
I think I hallucinated it.